packages/pyshp/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: pyshp
+  version: 2.3.1
+  top-level:
+    - shapefile
+source:
+  url: https://files.pythonhosted.org/packages/98/2f/68116db5b36b895c0450e3072b8cb6c2fac0359279b182ea97014d3c8ac0/pyshp-2.3.1-py2.py3-none-any.whl
+  sha256: 67024c0ccdc352ba5db777c4e968483782dfa78f8e200672a90d2d30fd8b7b49
+test:
```

Comment:
```suggestion
requirements:
  run:
    cartopy
test:
```

Reply:
Thanks, that got me onto the right track